participant/views.py

```diff
@@ -1716,3 +1715,61 @@ def database_live_api_export(self, request):
         except Exception as e:
             LOGGER.error(f"Failed to fetch data from api ERROR: {e} and input fields: {request.data}")
             return Response({"message": f"API Response: {e}"}, status=status.HTTP_400_BAD_REQUEST)
+
+
+class SupportTicketV2ModelViewSet(GenericViewSet):
+    parser_class = JSONParser
+    queryset = SupportTicketV2.objects.all()
+    serializer_class = SupportTicketV2Serializer
+    pagination_class = CustomPagination
+
+    @http_request_mutation
+    def list(self, request):
+        queryset = self.get_queryset()
+        map_id = request.META.get('map_id')
+        queryset = queryset.filter(user_map=map_id)
+        page = self.paginate_queryset(queryset)
+        support_tickets_serializer = SupportTicketV2Serializer(page, many=True)
+        return self.get_paginated_response(support_tickets_serializer.data)
+
+    # API to retrieve a single object by its ID
+    def retrieve(self, request, pk=None):
+        queryset = self.get_queryset()
+        object = get_object_or_404(queryset, pk=pk)
+        serializer = self.get_serializer(object)
+        return Response(serializer.data)
+
+    # API to create a new object
+    def create(self, request):
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        object = serializer.save()
+        return Response(serializer.data, status=status.HTTP_201_CREATED)
+
+    # API to update an existing object by its ID
+    def update(self, request, pk=None):
+        queryset = self.get_queryset()
+        object = get_object_or_404(queryset, pk=pk)
+        serializer = self.get_serializer(object, data=request.data)
+        serializer.is_valid(raise_exception=True)
+        object = serializer.save()
+        return Response(serializer.data)
+
+    # API to delete an existing object by its ID
+    def destroy(self, request, pk=None):
+        queryset = self.get_queryset()
+        object = get_object_or_404(queryset, pk=pk)
+        object.delete()
+        return Response(status=status.HTTP_204_NO_CONTENT)
+
+    @http_request_mutation
```

Comment:
Remove or comment  filter_support_tickets for now and start working on support ticket filters API.